releasenotes/notes/drop-python-3.7-c71f8b6559beaf86.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    The minimum required Python version was raised to Python 3.8.
+    To use rustworkx, please ensure you are using Python >= 3.8.
+deprecations:
+  - |
+    Support for Python 3.7 has been dropped as of 0.14.0.
```

Comment:
The upgrade release note you wrote covers this, we don't need a second one here:

```suggestion
```